siibra/api_call.m

```diff
@@ -0,0 +1,13 @@
+function result = api_call(uri_str)
```

Comment:
Suggest this function belongs within a `+internal` package/subpackage; possibly as a Static method of an internally packaged class

Reply:
I agree. The `webread` function, however, makes the `api_call` function obsolete. So I will delete the `api_call` function.